devops/setup-gpg/README.md

```diff
@@ -0,0 +1,29 @@
+# Setup GPG composite Action
+
+This composite action will setup your github action to use a supplied gpg key.
+
+## Supported Platroms
+- GPG
+- rpmsign/rpm
+- debsign
+
+If you need a platform not listed above, please contact the maintainers!
+
+## Example Usage
+
+```yaml
+on: [push]
+
+jobs:
+  hello_world_job:
+    runs-on: ubuntu-latest
+    name: A job to say hello
+    steps:
+      - uses: actions/checkout@v4
+      - id: foo
+        uses: aerospike/shared-workflows/devops/setup-gpg@latest
```

Comment:
It might help to use tags with semantic versioning so users can receive upstream changes without causing breaking changes. Like creating a tag `v1` to represent the latest `1.x.x`

Reply:
so this is a shared repo. we will have to think about this to be honest.